src/v/cloud_storage/remote.cc

```diff
@@ -183,17 +183,10 @@ ss::future<download_result> remote::download_manifest(
         } catch (...) {
             eptr = std::current_exception();
         }
+        co_await client->shutdown();
```

Comment:
What happens when we go on the second iteration of the while loop if the client has been shut down (in case of retry)?

Reply:
it will reconnect